tests/test_template_types.py

```diff
@@ -817,30 +879,48 @@ def test_subject_line_gets_applied_to_correct_template_types():
         LetterPreviewTemplate,
         LetterImageTemplate,
     ]:
-        assert issubclass(cls, WithSubjectTemplate)
+        assert issubclass(cls, SubjectMixin)
     for cls in [
         SMSMessageTemplate,
         SMSPreviewTemplate,
     ]:
-        assert not issubclass(cls, WithSubjectTemplate)
+        assert not issubclass(cls, SubjectMixin)
 
 
-def test_subject_line_gets_replaced():
-    template = WithSubjectTemplate({"content": '', 'subject': '((name))'})
+@pytest.mark.parametrize('template_class, template_type', (
+    (EmailPreviewTemplate, 'email'),
+    (LetterPreviewTemplate, 'letter'),
```

Comment:
Should we also test the other letter and email classes here? e.g. `LetterPrintTemplate`. Similar question for the other tests around this which are only testing the preview classes

Reply:
Added the extra test cases in a5eecb8